src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -267,11 +269,36 @@ public void setEt3ResponseDueDate(CaseData caseData) {
         }
     }
 
+    /**
+     * Update Work Allocation specific field.
+     *
+     * @param errors - if there are errors do not update the field
+     * @param caseData - the caseData contains the values for the case
+     */
+    public void updateWorkAllocationField(List<String> errors, CaseData caseData) {
+        if (errors.isEmpty()
+                && featureToggleService.isWorkAllocationEnabled()
+                && !isEmpty(caseData.getRespondentCollection())) {
+            updateResponseReceivedCounter(caseData.getRespondentCollection());
+        }
+    }
+
+    private void updateResponseReceivedCounter(List<RespondentSumTypeItem> respondentCollection) {
+        RespondentSumType firstRespondent = respondentCollection.get(0).getValue();
+        if (YES.equals(firstRespondent.getResponseReceived())) {
+            firstRespondent.setResponseReceivedCount(
+                    StringUtils.isBlank(firstRespondent.getResponseReceivedCount())
+                            ? "1"
```

Comment:
Avoid use of a magic number and use a descriptive constant

Reply:
Excellent point, updated.